.github/scripts/main.sh

```diff
@@ -20,6 +20,7 @@ export OCAMLRUNPARAM=b
   fi
 
   ./configure --prefix ~/local --with-mccs
+  echo 'DUNE_PROFILE=dev' >> Makefile.config
```

Comment:
One possibility might to do this _only_ when `"$OPAM_TEST" != "1"` - i.e. test the build with the dev profile but test the release profile version with opam-rt. @rjbou?

Reply:
Discussed this morning, and yes we'll do this!